### ISSUE TITLE
[TW-680] default command takes no arguments

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#715]
_Uli Martens on 2011-09-23T10:00:20Z says:_ 

 running task with a default command set to a single report runs fine:

<pre>
youam(at)riona:.../task (git)-[2.0.0]-$ task
[task unblocked]

ID  Deps Project                 Pri Due      Active Age  Tags        Description
[....]
</pre>

But when I change the default command to (at)unblocked prio.not:L@, task shows me that it knows this to be the wanted command, but doesn't execute it:
<pre>
youam(at)riona:.../task (git)-[2.0.0]-$ task
[task unblocked prio.not:L]

Usage: task                                      Runs rc.default.command, if specified.
       task active [<filter>]                    Lists active tasks matching the specified criteria
       task add <modifications>                  Adds a new task
[....]
</pre>

In the 1.9 branch, this worked as expected. In the 2.0 branch, I'm seeing this error for quite some time now. (I didn't investigate then because the snapshots I was using were to early)

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-09-23T10:00:20Z
Modified: 2014-02-09T02:06:33Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2011-09-23T11:57:40Z says:_ 

 Thanks.
